src/app/amsterdam/template.tsx

```diff
@@ -125,7 +125,7 @@ function Template({ children }) {
             </LinkList>
           </Header.GridCellNarrowWindowOnly>
           <Grid.Cell span="all">
-            <Heading className="ams-mb--sm" level={2} size="level-3">
```

Comment:
I've not mentioned this everywhere, but this is a recurring pattern. Do we not want to have spacing below Headings? It looks off, imo.